### PR TITLE
Use the odoo-sentinel library

### PR DESCRIPTION
**Pull request by guewen**
* Replace usage of the vendored library by odoo-sentinel from pypi
* Output sentinel's log file in stdout so we can see them from Docker
* Trap ctrl-c so we can stop a running container without resorting to `docker kill`
* Add 11.0 images
* Change release version to `11.0-x-y` where x is the odoo-sentinel release and y an increment for the release x

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,3 +81,17 @@ ssh -p 2222 scanneroperator@localhost
...
 You should see the ncurses client.
+
+## Release new image
+
+To release an updated image:
+
+* If the `odoo-sentinel` library has a new version, change the version in
+  `requirements.txt` and add a tag `x-1` (for instance 1.1-1 if the new
+  `odoo-sentinel` version is 1.1)
+* If you need a new build and the `odoo-sentinel` version did not changed, add
```
Comment (leemannd) [inline README.md]: s/changed/change/

**Review by leemannd — CHANGES_REQUESTED** [review]
Only nitpicking on the text.
Reviewed, but not sure to know all about the changes.

File: README.md (hunk 1 above)
Comment (guewen) [reply README.md]: fixed

**Review by leemannd — APPROVED** [review]

**Comment by simahawk** [comment]
well, mine was a :+1: :smile:

**Pull request merged**